### PR TITLE
#39 SMIB

### PR DESCRIPTION
**Pull request by MarianneSaugierRTE**

### CODE REVIEW

File: nrt/data/SMIB/RegulationProportional/SMIB_1_StepPm/SMIB.dyd [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    <dyn:unitDynamicModel id="busInf" name="Dynawo.Electrical.Buses.InfiniteBus" parFile="SMIB.par" parId="1"/>
+    <dyn:unitDynamicModel id="line1" name="Dynawo.Electrical.Lines.Line" parFile="SMIB.par" parId="2"/>
+    <dyn:unitDynamicModel id="line2" name="Dynawo.Electrical.Lines.Line" parFile="SMIB.par" parId="3"/>
+    <dyn:unitDynamicModel id="tfo" name="Dynawo.Electrical.Transformers.TransformerFixedRatio" parFile="SMIB.par" parId="4"/>
+    <dyn:connect id1="busInf" var1="terminal" id2="line1" var2="terminal1"/>
+    <dyn:connect id1="busInf" var1="terminal" id2="line2" var2="terminal1"/>
+    <dyn:connect id1="line1" var1="terminal2" id2="tfo" var2="terminal1"/>
+    <dyn:connect id1="line2" var1="terminal2" id2="tfo" var2="terminal1"/>
+  </dyn:modelicaModel>
+  <!-- Generator with four windings with fixed excitation voltage and proportional speed regulation -->
```
Comment (adriguir) [inline nrt/data/SMIB/RegulationProportional/SMIB_1_StepPm/SMIB.dyd]: It seems that both the speed regulation and the excitation voltage are proportional regulations

File: nrt/data/SMIB/RegulationProportional/SMIB_2_StepEfd/SMIB.dyd [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    <dyn:unitDynamicModel id="busInf" name="Dynawo.Electrical.Buses.InfiniteBus" parFile="SMIB.par" parId="1"/>
+    <dyn:unitDynamicModel id="line1" name="Dynawo.Electrical.Lines.Line" parFile="SMIB.par" parId="2"/>
+    <dyn:unitDynamicModel id="line2" name="Dynawo.Electrical.Lines.Line" parFile="SMIB.par" parId="3"/>
+    <dyn:unitDynamicModel id="tfo" name="Dynawo.Electrical.Transformers.TransformerFixedRatio" parFile="SMIB.par" parId="4"/>
+    <dyn:connect id1="busInf" var1="terminal" id2="line1" var2="terminal1"/>
+    <dyn:connect id1="busInf" var1="terminal" id2="line2" var2="terminal1"/>
+    <dyn:connect id1="line1" var1="terminal2" id2="tfo" var2="terminal1"/>
+    <dyn:connect id1="line2" var1="terminal2" id2="tfo" var2="terminal1"/>
+  </dyn:modelicaModel>
+  <!-- Generator with four windings with fixed excitation voltage and proportional speed regulation -->
```
Comment (adriguir) [inline nrt/data/SMIB/RegulationProportional/SMIB_2_StepEfd/SMIB.dyd]: Same remark as above

File: nrt/data/SMIB/RegulationProportional/SMIB_2_StepEfd/SMIB3.dyd [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    <dyn:initConnect id1="gen" var1="Efd0Pu" id2="efd" var2="Value0"/>
+    <dyn:connect id1="gover" var1="PmRefPu" id2="Pm" var2="setPoint"/>
+    <dyn:connect id1="gen" var1="PmPu" id2="gover" var2="PmPu"/>
+    <dyn:connect id1="gen" var1="omegaPu" id2="gover" var2="omegaPu"/>
+    <dyn:connect id1="gen" var1="efdPu" id2="efd" var2="step"/>
+    <dyn:connect id1="gen" var1="omegaRefPu" id2="omegaRef" var2="setPoint"/>
+  </dyn:modelicaModel>
+  <!-- Connections -->
+  <dyn:connect id1="Network" var1="tfo_terminal2" id2="SM" var2="gen_terminal"/>
+</dyn:dynamicModelsArchitecture>
```
Comment (adriguir) [inline nrt/data/SMIB/RegulationProportional/SMIB_2_StepEfd/SMIB3.dyd]: I haven't found a job using this dyd file at first glance

File: nrt/data/SMIB/RegulationProportional/SMIB_3_LoadVarQ/SMIB.dyd [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    <dyn:connect id1="line1" var1="terminal2" id2="tfo" var2="terminal1"/>
+    <dyn:connect id1="line2" var1="terminal2" id2="tfo" var2="terminal1"/>
+  </dyn:modelicaModel>
+  <!-- Load with reactive power variation-->
+  <dyn:modelicaModel id="Load">
+    <dyn:unitDynamicModel id="load" name="Dynawo.Electrical.Loads.LoadAlphaBeta" initName = "Dynawo.Electrical.Loads.Load_INIT" parFile="SMIB.par" parId="5"/>
+    <dyn:unitDynamicModel id="QRefPu" name="Dynawo.Electrical.Controls.Basics.Step" parFile="event.par" parId="1"/>
+    <dyn:connect id1="QRefPu" var1="step" id2="load" var2="QRefPu"/>
+  </dyn:modelicaModel>
+  <!-- Generator with four windings with fixed excitation voltage and proportional speed regulation -->
```
Comment (adriguir) [inline nrt/data/SMIB/RegulationProportional/SMIB_3_LoadVarQ/SMIB.dyd]: Same remark as above

File: nrt/data/SMIB/RegulationProportional/SMIB_4_DisconnectLine/SMIB.dyd [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+    <dyn:unitDynamicModel id="busInf" name="Dynawo.Electrical.Buses.InfiniteBus" parFile="SMIB.par" parId="1"/>
+    <dyn:unitDynamicModel id="line1" name="Dynawo.Electrical.Lines.Line" parFile="SMIB.par" parId="2"/>
+    <dyn:unitDynamicModel id="line2" name="Dynawo.Electrical.Lines.Line" parFile="SMIB.par" parId="3"/>
+    <dyn:unitDynamicModel id="tfo" name="Dynawo.Electrical.Transformers.TransformerFixedRatio" parFile="SMIB.par" parId="4"/>
+    <dyn:connect id1="busInf" var1="terminal" id2="line1" var2="terminal1"/>
+    <dyn:connect id1="busInf" var1="terminal" id2="line2" var2="terminal1"/>
+    <dyn:connect id1="line1" var1="terminal2" id2="tfo" var2="terminal1"/>
+    <dyn:connect id1="line2" var1="terminal2" id2="tfo" var2="terminal1"/>
+  </dyn:modelicaModel>
+  <!-- Generator with four windings with fixed excitation voltage and proportional speed regulation -->
```
Comment (adriguir) [inline nrt/data/SMIB/RegulationProportional/SMIB_4_DisconnectLine/SMIB.dyd]: Same as above

File: nrt/data/SMIB/RegulationProportional/SMIB_5_Fault/SMIB.dyd [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+    <dyn:unitDynamicModel id="line2" name="Dynawo.Electrical.Lines.Line" parFile="SMIB.par" parId="3"/>
+    <dyn:unitDynamicModel id="tfo" name="Dynawo.Electrical.Transformers.TransformerFixedRatio" parFile="SMIB.par" parId="4"/>
+    <dyn:unitDynamicModel id="fault" name="Dynawo.Electrical.Events.NodeFault" parFile="event.par" parId="1"/>
+    <dyn:connect id1="busInf" var1="terminal" id2="line1" var2="terminal1"/>
+    <dyn:connect id1="busInf" var1="terminal" id2="line2" var2="terminal1"/>
+    <dyn:connect id1="line1" var1="terminal2" id2="tfo" var2="terminal1"/>
+    <dyn:connect id1="line2" var1="terminal2" id2="tfo" var2="terminal1"/>
+    <dyn:connect id1="tfo" var1="terminal1" id2="fault" var2="terminal"/>
+  </dyn:modelicaModel>
+  <!-- Generator with four windings with fixed excitation voltage and proportional speed regulation -->
```
Comment (adriguir) [inline nrt/data/SMIB/RegulationProportional/SMIB_5_Fault/SMIB.dyd]: Same as above

File: nrt/data/SMIB/RegulationProportional/SMIB_1_StepPm/SMIB.dyd (hunk 1 above)
Comment (MarianneSaugierRTE) [reply nrt/data/SMIB/RegulationProportional/SMIB_1_StepPm/SMIB.dyd]: fixed

File: nrt/data/SMIB/RegulationProportional/SMIB_2_StepEfd/SMIB3.dyd (hunk 3 above)
Comment (MarianneSaugierRTE) [reply nrt/data/SMIB/RegulationProportional/SMIB_2_StepEfd/SMIB3.dyd]: true: deleted

File: nrt/data/SMIB/RegulationProportional/SMIB_3_LoadVarQ/SMIB.dyd (hunk 4 above)
Comment (MarianneSaugierRTE) [reply nrt/data/SMIB/RegulationProportional/SMIB_3_LoadVarQ/SMIB.dyd]: fixed

File: nrt/data/SMIB/RegulationProportional/SMIB_5_Fault/SMIB.dyd (hunk 6 above)
Comment (MarianneSaugierRTE) [reply nrt/data/SMIB/RegulationProportional/SMIB_5_Fault/SMIB.dyd]: fixed

File: nrt/data/SMIB/RegulationProportional/SMIB_2_StepEfd/SMIB.dyd (hunk 2 above)
Comment (MarianneSaugierRTE) [reply nrt/data/SMIB/RegulationProportional/SMIB_2_StepEfd/SMIB.dyd]: fixed

File: nrt/data/SMIB/RegulationProportional/SMIB_4_DisconnectLine/SMIB.dyd (hunk 5 above)
Comment (MarianneSaugierRTE) [reply nrt/data/SMIB/RegulationProportional/SMIB_4_DisconnectLine/SMIB.dyd]: fixed

**Pull request merged**